README.md

```diff
@@ -1,48 +1,53 @@
-# robosub-ros-docker
+# Duke Robotics Club - RoboSub ROS Docker
+
 ![Build Status](https://github.com/dukerobotics/robosub-ros-docker/workflows/build/badge.svg)
 ![Docker Pulls](https://img.shields.io/docker/pulls/dukerobotics/robosub-ros)
 
-Docker image for ROS system to control a robot for the RoboSub Competition
+We are a Duke University club competing in the RoboSub Competition. Check out [our website](https://duke-robotics.com).
+
+This repo contains our Dockerfiles and instructions for how to build them, in support of our main [robosub-ros](https://github.com/DukeRobotics/robosub-ros) repo. If you're looking to use these images to run our code, check out the main repo.
+
+Our images are automatically built and pushed to [our Docker Hub repo](https://hub.docker.com/r/dukerobotics/robosub-ros) using GitHub CI/CD.
```

Comment:
Technically, the name is Github Actions, CI/CD usually refers to Gitlab.

Also, is it Docker Hub or Dockerhub or dockerhub? (Git Hub vs Github vs github).

Reply:
Made the change.

It's actually Docker Hub. I was surprised too. [https://hub.docker.com/](https://hub.docker.com/)